docs/user/metrics/datetime.md

```diff
@@ -176,6 +176,36 @@ Assert.Equal(1, Install.firstRun.TestGetNumRecordedErrors(ErrorType.InvalidValue
 
 </div>
 
+<div data-lang="Rust" class="tab">
+
+```Rust
+use glean::ErrorType;
+use glean_metrics;
+
+use chrono::{FixedOffset, TimeZone};
+
+// Records "now"
+install::first_run.set(None);
+// Records a custom datetime
+install::first_run.set_with_details(2019, 3, 25, 0, 0, 0, 0);
+```
+
+There are test APIs available too.
+
+```Rust
+// Was anything recorded?
+assert!(metrics.install.first_run.test_get_value(None).is_some());
+
+// Was it the expected value?
+assert_eq!("2019-03-25-05:00", metrics.install.first_run.test_get_value(None));
```

Comment:
Does comparing a datetime to a string actually work?  (I'm not saying it doesn't, but the test below doesn't do that ... maybe we should add that to the test to make sure this example works...)

Reply:
Whoops, nope, the docs are wrong. Good catch.